src/actions/WEBHOOKS_UPDATE.ts

```diff
@@ -0,0 +1,21 @@
+import { Action } from '../lib/structures/Action';
+
+import type { WebhooksUpdateDispatch } from '@klasa/ws';
+import type { TextChannel } from '../client/caching/structures/channels/TextChannel';
+import type { NewsChannel } from '../client/caching/structures/channels/NewsChannel';
+
+export default class CoreAction extends Action {
+
+	public check(data: WebhooksUpdateDispatch): TextChannel | NewsChannel | null {
+		return this.client.guilds.get(data.d.guild_id)?.channels.get(data.d.channel_id) as TextChannel | NewsChannel ?? null;
+	}
+
+	public build(): null {
+		return null;
```

Comment:
Shouldn't the webhook be rebuilt here? I mean, the details would've changed for this to emit

Reply:
There's no webhook data, it just emits `guild_id` and `channel_id`.